source/examples.rst

```diff
@@ -9,4 +9,12 @@ Text to image GUI
 
 See : `EMF Demos - Text to Image GUI <https://github.com/easy-model-fusion/demos/tree/main/text-to-image-gui>`_
 
-The code defines a graphical user interface (GUI) application using the Tkinter library. This application allows users to generate images from text using the `Stabilityai - Stable Diffusion XL Base 1.0 <https://huggingface.co/stabilityai/stable-diffusion-xl-base-1.0>`_ model.
\ No newline at end of file
+The code defines a graphical user interface (GUI) application using the Tkinter library. This application allows users to generate images from text using the `Stabilityai - Stable Diffusion XL Base 1.0 <https://huggingface.co/stabilityai/stable-diffusion-xl-base-1.0>`_ model.
+
+
+Images from Text Prompts Generated by Text Model GUI
```

Comment:
might be more natural to reverse the order ?
i am assuming a user opening the doc searching for a specific example would first be looking at the input and then the result ?

Reply:
Title updated